independent-publisher-connectors/SolarEdge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2062 @@
+                          }
+                        }
+                      },
+                      "description": "telemetries"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/equipment/{site_id}/{equipment_id}/changeLog": {
+      "get": {
+        "summary": "Equipment Change Log",
+        "description": "Returns a list of equipment component replacements ordered by date. This method is applicable to inverters, optimizers, batteries and gateways.",
+        "operationId": "EquipmentChangeLog",
+        "parameters": [
+          {
+            "x-ms-url-encoding": "single",
+            "name": "site_id",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "description": "The site identifier",
+            "x-ms-summary": "The site identifier",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "equipment_id",
+            "x-ms-url-encoding": "single",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The serial number of the equipment",
+            "x-ms-summary": "The serial number of the equipment",
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ChangeLog": {
+                  "type": "object",
+                  "properties": {
+                    "count": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "count"
+                    },
+                    "list": {
+                      "type": "object",
+                      "properties": {
+                        "serialNumber": {
+                          "type": "string",
+                          "description": "Equipment short serial number"
+                        },
+                        "partNumber": {
+                          "type": "string",
+                          "description": "inverter/battery/optimizer/gateway model"
+                        },
+                        "date": {
+                          "type": "string",
+                          "description": "Date of replacement of that equipment component"
+                        }
+                      },
+                      "description": "list of equipment"
+                    }
+                  },
+                  "description": "ChangeLog"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {},
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.solaredge.com/"
```

Comment:
@RichardWie please provide a valid URL 
![image](https://github.com/microsoft/PowerPlatformConnectors/assets/97551789/f67d0639-e0b9-44b2-8db7-04fbcdfdb7c3)

Reply:
URL is working from my side, this is the URL I get from a search engine too